release_dashboard/lib/widgets/engine_cherrypicks_substeps.dart

```diff
@@ -2,14 +2,24 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import '../state/status_state.dart';
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
 
 import 'common/checkbox_substep.dart';
+import 'common/url_button.dart';
 
 enum EngineCherrypicksSubstep {
-  substep1,
-  substep2,
-  substep3,
+  verifyRelease,
+  applyCherrypicks,
+  resolveConflicts,
```

Comment:
It seems like applyCherrypicks and resolveConflicts are the same step. Can they be deduplicated?

Reply:
You are right. Done!